assets/i18n/zh-tw.json

```
@@ -0,0 +1,269 @@
+{
+    "@metadata": {
+        "authors": [
+            "tw-dog",
+            "getlocalization contributors",
+            "etc"
+        ],
+        "last-updated": "2021-05-21",
+        "locale": "zh-tw",
+        "message-documentation": "qqq",
+        "website-url": "dogecoin.com"
+    },
+    "dgc-title": "狗狗幣",
+    "dgc-title-firstcap": "狗狗幣", 
+    "dgc-title-caps": "狗狗幣", 
+    "dgc-alt-watch-video": "看影片", 
+    "dgc-alt-logo": "狗狗幣 圖標", 
+    "dgc-header-desc": "狗狗幣是一種開源的加密貨幣，全世界的狗狗都很喜歡。", 
```

Original sentence "Dogecoin is an open source peer-to-peer digital currency, favored by Shiba Inus worldwide."
```suggestion
    "dgc-header-desc": "狗狗幣是一種開源的加密貨幣，深獲全世界柴犬們的喜愛。", 
```
This might be more accurate.